registry/job.go

```diff
@@ -127,18 +130,14 @@ func (r *EtcdRegistry) Units() ([]job.Unit, error) {
 		if u == nil {
 			continue
 		}
-		uMap[u.Name] = u
-	}
-
-	var sortable sort.StringSlice
-	for name, _ := range uMap {
-		sortable = append(sortable, name)
-	}
-	sortable.Sort()
+		// Just a guard
+		_, ok := uMap[u.Name]
+		if ok {
+			continue
+		}
```

Comment:
Actually as stated in the commit log, we don't really get dupes but even if we did the old code was filtering them inside a hash, thus we always end up with only one unit name in the final slice returned to the caller., but now theoretically with this change and without that guard we may end up with two same names in the returned slice since the append to slice was moved... and as stated in the commit log and commented this is just a guard, I can remove it.